clang/test/3C/basic.c

```diff
@@ -183,7 +183,7 @@ struct student *new_student() {
 }
 //CHECK: _Ptr<struct student> new_student(void) {
 //CHECK_NOALL: char name[] = "Bilbo Baggins";
-//CHECK_ALL: char name _Nt_checked[14] =  "Bilbo Baggins";
```

Comment:
Is this behavior preferred? Seems OK to me, I'm just wondering why it happened.

Reply:
Yeah, I wasn't entirely sure about this either, but it seems to be fine as far as  CheckedC is concerned.  I thought about checking for this case to fall back to the old behavior, but that wouldn't be straight forward. In the AST, the bounds are defined as the actual bounds of the array, so I coulnd't find a method in clang API to check directly. Checking based on the source string would work for the nice case, but if there's a space between the brackets (or a comment, or a macro, etc.) the simple check doesn't work anymore.